src/lib.rs

```diff
@@ -14,7 +15,7 @@ use diesel::query_builder::{QueryFragment, QueryId};
 /// All queries that require more time than the specified value
 /// are logged.
 pub struct LoggingConnection<C: Connection> {
-    conn: C,
+    pub conn: C,
```

Comment:
Since this is a public library which is meant to be used with another library, it would be good to have a comment `//` on why `conn` is now public (convenience, necessity, etc).